starter-code/routes/passportRouter.js

```diff
@@ -14,9 +14,54 @@ router.get("/private-page", ensureLogin.ensureLoggedIn(), (req, res) => {
   res.render("passport/private", { user: req.user });
 });
 
+router.get("/signup", (req, res, next) => {
+  res.render("passport/signup");
+});
+
+router.post("/signup", (req, res, next) => {
+  const username = req.body.username;
+  const password = req.body.password;
+
+  if (username === "" || password === "") {
+    res.render("auth/signup", { message: "Indicate username and password" });
```

Comment:
It looks like instead of `"auth/signup"`, it should be `"passport/signup"`. I got an error when I tried to run it.